testing-endpoints.md

```diff
@@ -18,20 +18,60 @@ curl -X GET http://localhost:8080/api/test/did-debug
 ```
 
 
+#### `api/test/generate-did`
+
+Returns a Long-Form did made from input public keys in jwk format
+
+The body is a JSON object (note, this API requires a content-type header) with the following fields:
+1. `recoveryPublicJwk` - **required** `ES256` public signing key jwk for did recovery operations
+2. `updatePublicJwk` - **required** `ES256` public signing key jwk for did update operations
+3. `signingPublicJwk` - **optional** `ES256` public signing key jwk for verifying assertions 
+4. `encryptionPublicJwk` - **optional** public encryption key jwk using the `"alg": "ECDH-ES"` and `"enc": "A256GCM"` encryption algorithm
+5. `domains` - **optional** array of string URI hostnames. Will be added to the did's `LinkedDomains` by an `add-services` action. 
+
+**Note**: The `signingPublicJwk` and `encryptionPublicJwk` fields are optional since 
```

Comment:
While it's technically true that sidetree doesn't require a DID Document to include signing and encryption keys, the health cards spec does:

> The response is signed as a JWS with the user's DID and optionally encrypted using the lab's DID (if the request specified id_token_encrypted_response_*). The latter step requires looking inside the DID Document for an encryption key, which can be used for encrypting a payload for this party.

(https://healthwallet.cards/#connect-health-wallet-to-lab-account)

As such, my 2¢ is to require the signing key to be present -- just to facilitate those getting started with health cards to have as few roadblocks as possible.

Reply:
I'm on board with this. I'll work on making this required in 1) the documentation, 2) the API, 3) the internal function called